style.css

```diff
@@ -315,55 +331,82 @@ body {
   padding: 20px;
 }
 
-h2.contact-me {
-  text-align: center;
-}
-
 .ContactDesctiption {
   margin-bottom: 20px;
+  font-family: "Poppins", sans-serif;
+  font-size: 20px;
+  text-align: center;
+  color: hsl(240, 100%, 96%);
 }
 
 form {
   display: flex;
   flex-direction: column;
+  font-family: 'Poppins', sans-serif;
 }
 
 .form-group {
   margin-bottom: 20px;
 }
 
-label {
-  font-weight: bold;
-}
-
 input,
 textarea {
   width: 100%;
   padding: 10px;
-  border: 1px solid #ccc;
-  border-radius: 4px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 17px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  color: hsl(218, 54%, 20%);
+  background: hsl(0, 0%, 100%);
+  border: 2px solid hsl(198, 16%, 84%);
 }
 
 textarea {
   height: 150px;
 }
 
+.btntouch {
+  margin-bottom: 20px;
+  text-align: center;
+}
+
+.continer {
+  text-align: center;
+}
+
 .btnGettouch {
-  display: flex;
-  justify-content: center;
   padding: 10px 15px;
   border-radius: 8px;
   color: hsla(234, 100%, 69%, 1);
   background-color: hsl(0, 0%, 100%);
-  outline: none;
   border: 1px solid #396df2;
   width: fit-content;
   cursor: pointer;
   font-size: 17px;
   font-weight: 600;
-  margin-left: 5px;
+  margin-right: auto;
+  display: block;
 }
 
 .btnGettouch:hover {
-  background-color: hsl(235, 30%, 93%);
+  background: hsla(234, 100%, 69%, 1);
+  color: hsl(0, 0%, 100%);
+}
+
+.btnGettouch:active {
+  background: hsl(0, 0%, 100%);
+  color: linear-gradient(0deg, hsl(0, 0%, 100%), hsl(0, 0%, 100%)), linear-gradient(0deg, hsl(234, 100%, 75%), hsl(234, 100%, 75%));
+}
+
+.btnGettouch:disabled {
+  background-color: hsl(0, 0%, 80%);
+  color: hsl(218, 17%, 44%);
+}
+
+.myinput::placeholder {
+  color: hsla(217, 13%, 74%, 1);
```

Comment:
You did well implementing the changes requested previously, but you didn't implement the first and fourth suggestion which is 
 > Please I'd suggest you add some margin between the About  section and the Contact section so as to make it visually appealing 😄 ,
and 
>  I'd suggest you use the given placeholder color as it is given in the Figma design so it looks a bit better than what you have

Please implement the change so as to have a better design

Figma design :
![image](https://github.com/melkamu12/Portfolio-PairProgramming/assets/20537712/cfed354a-7f76-482c-81cc-76ab96697cc1)

Your design :

![image](https://github.com/melkamu12/Portfolio-PairProgramming/assets/20537712/4fabbc7b-5416-40f9-bf42-426bd171c95c)